### PR TITLE
Bookings controller

### PR DESCRIPTION
**Pull request by lafaust**
I added a controller for bookings.
No views yet.

### CODE REVIEW

**Comment by Nschmitt83** [comment]
Yes, but can we know why @lafaust ?! The views are very importants you know...

**Comment by lafaust** [comment]
> Yes, but can we know why @lafaust ?! The views are very importants you know...

lol "not YET done"...

File: app/models/user.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class User < ApplicationRecord
-  has_many :reviews, :bookings, :guitars
```
Comment (Nschmitt83) [inline app/models/user.rb]: Peut on savoir ce que cela signifie ?

Comment (Nschmitt83) [reply app/models/user.rb]: je déconnais. J'avais très bien compris!

**Pull request merged**